AirCasting/CoreData/Session.swift

```diff
@@ -135,16 +135,17 @@ enum StreamingMethod: Int {
 
 public enum DeviceType: Int, CustomStringConvertible {
     case MIC = 0
-    case AIRBEAM3 = 1
-    case AIRBEAMMINI = 2
+    case AIRBEAM = 1
 
     public var description: String {
         switch self {
         case .MIC: return "Device's Microphone"
-        case .AIRBEAM3: return "AirBeam 3"
-        case .AIRBEAMMINI: return "AirBeamMini"
+        case .AIRBEAM: return "AirBeam"
         }
     }
+    func isAirBeam() -> Bool {
```

Comment:
This seems a little redundant, I don't think I'm getting what is the upside of calling 
`deviceType.isAirBeam() 
`
over doing 
`deviceType == .AIRBEAM 
`

Reply:
True :D it was created before for different thing and we should have remove it :D